### PR TITLE
create todo-list structure

### PR DESCRIPTION
**Pull request by huzaifakhan0308**
- create a Todo-list structure

### CODE REVIEW

**Review by andersonlebon — COMMENTED** [review]
## Hi @huzaifakhan0308 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive pull request, does not contain a [READEME](https://github.com/microverseinc/readme-template), and does not have linters. Kindly work on these before requesting your next review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by roshan-bajgain — COMMENTED** [review]
Hi @huzaifakhan0308, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a linter error. Please fix the error and submit it again

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @huzaifakhan0308, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/obj.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default class Obj {
+  constructor(NewValue, index) {
+    this.inputValue = NewValue;
+    this.indexNumber = index;
+    this.completed = true;
+  }
```
Comment (ambrose-kibet) [inline src/modules/obj.js:6]: I sugest keepin the task completed false  by default. just because it less confusing to add a completed task to a todo list.

**Pull request merged**